bob-common/src/node.rs

```diff
@@ -46,6 +46,7 @@ struct NodeInner {
 pub struct Output<T> {
     node_name: NodeName,
     inner: T,
+    affected_replicas: usize
```

Comment:
I don't like that this field is now in the results of all operations, although it is only needed for the put operation.
There are 2 possible ways to avoid that:
1. Change `PutResult` to something like `Result<NodeOutput<PutResponse>, NodeOutput<PutError>>`, where `PutResponse` will contain `affected_replicas` and `PutError` will contain `affected_replicas` + `error`.
2. Perform similar changes, but on the `operations.rs` level. Declare `RemotePutError` and `RemotePutResponse` in `support_types.rs`, add `affected_replicas` to these structures, and convert put result from `BobClient` to these types (something similar is done right now for `delete` where regular error is converted to `RemoteDeleteError`)

Personally, I'd prefer the second way, because in this way the network code is not affected